compiler/rustc_lint/src/internal.rs

```diff
@@ -267,3 +267,47 @@ impl EarlyLintPass for LintPassImpl {
         }
     }
 }
+
+declare_tool_lint! {
+    pub rustc::EXISTING_DOC_KEYWORD,
+    Allow,
```

Comment:
why are you using `Allow` here, I personally expect `Warn` to be more appropriate here.

Reply:
Because it's run outside of `rustc` apparently, so non-rustc things don't care about this lint (quite the opposite in fact since it's supposed to be used on non-existing keywords in case of proc-macros).

Reply:
yeah, but even if a lint pass is `Allow` it still runs, it just emits nothing if it's triggered, which doesn't happen outside of std

Reply:
> quite the opposite in fact since it's supposed to be used on non-existing keywords in case of proc-macros

can you elaborate here

Reply:
I did in [this comment](https://github.com/rust-lang/rust/pull/79541#discussion_r535553427) just above.